esp/src/eclwatch/ESPWorkunit.js

```diff
@@ -266,6 +266,12 @@ define([
         isDeleted: function () {
             return this.StateID === 999;
         },
+        isScheduled: function () {
+            return this.EventSchedule === 0 || this.EventSchedule === 1 ? true : false;
+        },
+        isNotScheduled: function () {
+            return this.EventSchedule === 0 || this.EventSchedule === 2 ? true : false;
+        },
```

Comment:
From the ESPWorkunit API point of view it might be cleaner to have:
hasScheduled + isScheduled methods which return this.EventSchedule !== 0 and this.EventSchedule === 1 respectively and then in the page display you can have "wu.wuScheduled && wu.isScheduled" for enabling / disabling the button.  (as it stands the API call could be misleading in a different context).